Gems/Multiplayer/Code/Source/Components/LocalPredictionPlayerInputComponent.cpp

```diff
@@ -333,8 +333,10 @@ namespace Multiplayer
                 }
  #endif
 
-                // Send correction
-                SendClientInputCorrection(m_lastClientInputId, correction);
+                // Send correction. Include both the latest client input host frame id and the latest client input id processed so that
+                // the client can ensure that it doesn't try to process out-of-order corrections. The client input id by itself
+                // can roll over its value too quickly to be useful for detecting out-of-order conditions.
```

Comment:
Maybe explain a bit further about ClientInputId is uint16_t, and so this value rolls over to 0 again if it reaches 2^16, which at 30 inputs a second, can happen after ~2000 seconds. etc etc

Reply:
Sure thing! I've put those comments in other places already, but no harm in having it here too.

Reply:
Changing it to this:
```
                // Send correction. Include both the latest client input host frame id and the latest client input id processed so that
                // the client can ensure that it doesn't try to process out-of-order corrections. The client input id is a uint16, which
                // can roll over in (65536 / 60 fps) which is < 20 minutes. If half that time or more passes between corrections,
                // so if we only tried to rely on the client input id to detect out-of-order corrections, we wouldn't be able to tell if
                // the difference is telling us that it's out of order or if a long time had passed. By sending the host frame id too,
                // we can distinguish between the two cases.
```